Breeze.UI/src/app/wallet/history/history.component.html

```diff
@@ -22,25 +22,11 @@ <h5 class="text-uppercase text-center">Search for a transaction</h5>
 
     <!-- TRANSACTIONS -->
     <section id="transaction" class="col-10 mx-auto">
-      <h5 class="mt-4">Transactions</h5>
+      <h5 *ngIf="!isSearching" class="mt-4">Displaying all transactions</h5>
+      <h5 *ngIf="transactions.length === 1 && isSearching" class="mt-4">Displaying one transaction</h5>
```

Comment:
Is it supposed to be `!isSearching`?

Reply:
I believe the code is correct here. If they are searching and one result then display one result text, else we show default.